eng/CadlEmitterUT.ps1

```diff
@@ -0,0 +1,11 @@
+#Requires -Version 7.0
```

Comment:
Can we call this file  ExecuteCadlEmitterUnitTests which is more consistent with the existing script

Reply:
updated